src/ResourceManager/AnalysisServices/Commands.AnalysisServices.ServiceManagement/AzureRM.AnalysisServices.ServiceManagement.psd1

```diff
@@ -0,0 +1,106 @@
+﻿#  
```

Comment:
@markcowl ? Dont get it. This is a separate project and separate module. how will the dll be packaged if I remove this.